### ISSUE TITLE
setup eslint to report error if importing the top level index.ts

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#873]
setup eslint to report error if importing the top level index.ts

**Comment by g-back** [comment]
https://www.npmjs.com/package/eslint-plugin-import

**Comment by larshp** [comment]
yes

**Comment by larshp** [comment]
argh, keep running into this

no-cycle: there are cycles, in core run `npm run madge.circular`

**Comment by larshp** [comment]
https://github.com/benmosher/eslint-plugin-import/blob/HEAD/docs/rules/no-restricted-paths.md might work, we can move the index.ts to a specific folder

**Comment by g-back** [comment]
Yes, I tested activating `no-cycles` and it give a couple of thousand findings
